recipes/configs/qwen2/0.5B_lora.yaml

```diff
@@ -65,20 +66,21 @@ loss:
 # Training
 epochs: 1
 max_steps_per_epoch: null
-gradient_accumulation_steps: 32
+gradient_accumulation_steps: 4
 
 # Logging
 output_dir: /tmp/Qwen2-0.5B-Instruct-lora-finetune
 metric_logger:
   _component_: torchtune.utils.metric_logging.DiskLogger
   log_dir: ${output_dir}
+
 log_every_n_steps: 1
 log_peak_memory_stats: False
 
 # Environment
 device: cuda
 dtype: bf16
-enable_activation_checkpointing: False
+enable_activation_checkpointing: True
```

Comment:
We need activation checkpointing now?

Reply:
I think the tradeoff for double batch size was net positive, but I can double check